bignum/boolean.awk

```diff
@@ -1,48 +1,73 @@
 # Boolean replacement functions
 
-function lshift(num, times){
```

Comment:
The reason `lshift`, `rshift`, `and`, `or`, `xor`, `not`, were located in a different file and named that way, was to work as replacements of gawk's bit functions when needed. However if you rename them, then they'll be always needed, and they are way slower than the native ones.

Reply:
Your idea of placing bitwise operations functions in a separate file was initially clear to me and I fully agree with you about the performance of the built-in gawk functions. But when using the library on a pre-unknown awk implementation, we get portability issues, because when you try to override a built-in function, gawk will exit and generate an error. The ideal solution would be to declare a function only if it does not exist, or if the script is not running on gawk. I have not found a way to implement any of this, so if you know how to implement one of this - please let me know, I will be very grateful to you.